PencilCase/PencilCase/scripts/Pages/Consts.ts

```diff
@@ -0,0 +1,11 @@
+﻿export let Pages = {
```

Comment:
Constants here.
Make sure the pageId in page view model and view (HTML, <div> with data-role equals to page) is identical to the Ids in this page.
The navigating system depends on them.